### PR TITLE
deprecate endorser binary

### PR DESCRIPTION
**Pull request by nicolasochem**
Mondaynet starts on Hangzhou then upgrades to alpha. Hangzhou has an
endorser but alpha does not. Currently there is no way to handle this.
This PR fixes it.

The endorser days are numbered, so instead of giving people to start or
not start an endorser based on the protocol, I just hardcoded this
simple logic:

  Start endorser if granada or hangzhou, else not.

### CODE REVIEW

**Comment by harryttd** [comment]
this can be eliminated in mkchain: https://github.com/oxheadalpha/tezos-k8s/blob/2f118ecb12/mkchain/tqchain/mkchain.py#L232

**Review by harryttd — APPROVED** [review]
Didn't test.  I assume it works

File: charts/tezos/templates/_containers.tpl [hunk 1]
```diff
@@ -234,24 +234,16 @@ https://github.com/helm/helm/issues/5979#issuecomment-518231758
...
       value: baker
-{{- end }}
-{{- end }}
-{{- end }}
-
-{{- define "tezos.container.endorsers" }}
-{{- if has "endorser" $.node_vals.runs }}
-{{ $node_vals_images := $.node_vals.images | default dict }}
-{{- range .Values.protocols }}
+{{- if or (regexFind "GRANAD" .command) (regexFind "Hangz" .command) }}
```
[the remark this answers is not in the document]
Comment (nicolasochem) [reply charts/tezos/templates/_containers.tpl:237]: Why is it desirable? Endorser days are numbered. Take this as interim code. After protocol I is on mainnet, we will just remove the entire if statement and remove all trace of endorsers in tezos-k8s code.

**Review by elric1 — APPROVED** [review]

**Comment by elric1** [comment]
I just remembered why I wanted the list: development. I'm currently running a test of tenderbake where we migrate to it. I need one endorser not two---but it is protocol alpha. In other tests, protocol alpha is older.

**Comment by nicolasochem** [comment]
We talked about this PR during the call today. There is no currently ongoing private chain effort which would require setting up an indexer in any other protocol.

But:
* the accuser is protocol specific and right now we are not spanning it at all
* in the future, rollups will have optional daemons

At the end, this PR satisfies people's known needs today. The configurability may be needed later, we should develop it when there is a use case.

**Pull request merged**